plugin/src/main/java/io/jenkins/plugins/casc/core/GlobalNodePropertiesConfigurator.java

```diff
@@ -0,0 +1,93 @@
+package io.jenkins.plugins.casc.core;
+
+import edu.umd.cs.findbugs.annotations.CheckForNull;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.Extension;
+import hudson.slaves.EnvironmentVariablesNodeProperty;
+import hudson.slaves.EnvironmentVariablesNodeProperty.Entry;
+import io.jenkins.plugins.casc.Attribute;
+import io.jenkins.plugins.casc.BaseConfigurator;
+import io.jenkins.plugins.casc.ConfigurationContext;
+import io.jenkins.plugins.casc.ConfiguratorException;
+import io.jenkins.plugins.casc.impl.attributes.MultivaluedAttribute;
+import io.jenkins.plugins.casc.model.CNode;
+import io.jenkins.plugins.casc.model.Mapping;
+import io.jenkins.plugins.casc.model.Sequence;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+@Extension
+public class GlobalNodePropertiesConfigurator extends BaseConfigurator<EnvironmentVariablesNodeProperty> {
+
+    @NonNull
+    @Override
+    public String getName() {
+        return "globalNodeProperties";
```

Comment:
are there other `globalNodeProperties`? and is there any impact to those from this change?

I see the test coverage here is only about env vars

Reply:
Yes, I'm calling `super.describe()` before checking `env` to make sure the of properties are still treated as they were, E.g. export works with the rest of global props.
```
globalNodeProperties:
  - "disableDeferredWipeout"
  - diskSpaceMonitor:
      freeDiskSpaceThreshold: "1GiB"
      freeDiskSpaceWarningThreshold: "2GiB"
      freeTempSpaceThreshold: "1GiB"
      freeTempSpaceWarningThreshold: "2GiB"
  - envVars:
      env:
      - key: "EMPTY_VAR"
      - key: "VARIABLE1"
        value: "foo"
  - toolLocation:
      locations:
      - home: "/home/myuser/bin/git"
        key: "hudson.plugins.git.GitTool$DescriptorImpl@Default"
```
More than happy to add an additional test for this anyway

Reply:
Why do you need to override it then as you aren't changing the export behaviour of `env`?

Reply:
Well seen, it's there because I first attempted to change the export. It's not really needed, removing the override (also added https://github.com/jenkinsci/configuration-as-code-plugin/pull/2657/commits/be60c5ab2bcab5df2b58344365848bcbabe9a9e5 with tests for the rest of global properties)